test/fixtures/spago-configWithBackend.dhall

```diff
@@ -0,0 +1,15 @@
+{-
+Welcome to a Spago project!
+You can edit this file as you like.
+-}
+{ backend =
+    "echo hi from backend> alternate-backend-output.txt"
```

Comment:
Oh this is clever 😄

Reply:
I would have like to also check, that no arguments are passed to the backend command, but `$@` is not supported on windows and causes the ci to fail